server/scsynth/SC_Lib_Cintf.cpp

```diff
@@ -283,17 +283,21 @@ bool checkServerVersion(void* f, const char* filename) {
 
 static bool PlugIn_Load(const bfs::path& filename) {
 #ifdef _WIN32
-    HINSTANCE hinstance = LoadLibraryW(filename.wstring().c_str());
+    HINSTANCE hinstance = LoadLibrary(filename.wstring().c_str());
```

Comment:
btw, i'd recommend to always use the `A` or `W` suffixed functions explicitly instead of relying on the preprocessor defines. this also allows you to mix ansi and wide apis in the same codebase

Reply:
Thanks for the feedback!
In that case I'd change all the calls to the Wide versions for consistency, including e.g. [FormatMessage](https://github.com/supercollider/supercollider/pull/6124/files#diff-f36a627946e1f45158f677041908350129be5deb4f0b82537a03c8751b143f67L293). Should I do this?

Reply:
not sure if it's worth the effort. when it's about obtaining error codes, the ansi variant of the functions is probably good enough. main rationale is not to rely on the `UNICODE` preprocessor define (e.g. never use `LoadLibrary`, but either `LoadLibraryW` or `LoadLibraryA`)

details: https://learn.microsoft.com/en-us/windows/win32/intl/conventions-for-function-prototypes

Reply:
The page you linked says:
> They [applications] should be written with generic functions, and should define UNICODE to compile the functions into Unicode functions.

So it seems that the MS's advice is to use generic functions and the preprocessor definition, right?

Reply:
i'd not follow microsoft's recommendation here, but be explicit (in my experience relying on the preprocessor define calls for trouble the more complex the codebase becomes)

Reply:
Updated. Thoughts?